.flake8

```diff
@@ -65,6 +65,7 @@ extend-ignore =
   N802    # FIXME: func name should be lowercase
   N816    # FIXME: mixed case var name
   P101    # FIXME: format string with unindexed params
+  PT007   # FIXME: wrong values type in @pytest.mark.parametrize, expected tuple
```

Comment:
@webknjaz Had to add this, otherwise `flake8` was reporting error.  I think it's a false positive.  I am using `parametrize` with `indirect=True`.  Here is an example usage:

```python
@pytest.mark.parametrize(
        '_setUp',
        [('test_modify_post_data_plugin')],
        indirect=True,
    )   # type: ignore[misc]
```

Reply:
No, it's not a false-positive. It is set up to expect a tuple for the param variants and you use a list in your example. It's a style I chose for consistency.
```diff
 @pytest.mark.parametrize(
         '_setUp',
-        [('test_modify_post_data_plugin')],
+        (
             ('test_modify_post_data_plugin'),
         ),
         indirect=True,
     )   # type: ignore[misc]
```

So drop the ignore and follow the selected style:
```suggestion
```
You could adjust the settings of this plugin later if you want.

Reply:
Thank you, lemme address this right in.